docs/prepare-image.md

```diff
@@ -0,0 +1,55 @@
+# Bootstraping a snappy device
+
+The `snap prepare-image` command is designed to do the snap specific
+part of building an image.
+
+## Running it
+
+The `snap prepare-image` command takes a model assertion and a rootdir
+as intput. It will create $ROOT/image that contains the directory layout
+with content for the image. It will also create $ROOT/gadget that
+contains the unpacked gadget snap content for ubuntu-image.
+
+It will also inspect the gadget snap for the bootloader
+configuration file and instlal that into the root-dir
```

Comment:
install*
“the root-dir” -> `$ROOT`? otherwise you're calling the same thing three different things, unless they are actually different and you haven't explained the difference